src/app/components/Buttons/index.ts

```diff
@@ -0,0 +1 @@
+export { PrimaryButton, IconButton } from "./Buttons";
```

Comment:
here we can just use * which means reexporting everything from the file
`export * from "./Buttons";`
p.s. overall I guess it would be better to have separate file for each btn

Reply:
Check it out, please, I pushed a new commit.

Reply:
I've just checked commit and I need to apologize.
I should have put more details.
I meant to leave Buttons folder as it is but create separate files for each button
and then reexport them via index.ts